dev/sbt-plugin/src/main/scala/com/lightbend/lagom/sbt/LagomPlugin.scala

```diff
@@ -339,8 +339,12 @@ object LagomPlugin extends AutoPlugin {
 
   private def dontAggregate(keys: Scoped*): Seq[Setting[_]] = keys.map(aggregate in _ := false)
 
+  private val cassandraPort: Int = 4000
+  private val cassandraStaticServices: Map[String, String] =
+    Map("cas_native" -> s"tcp://127.0.0.1:$cassandraPort/cas_native")
```

Comment:
Cassandra port is configurable (see `lagomCassandraPort`).  It shouldn't be hard coded here.  If you wanted this to be a separate definition (and not declared down where `lagomUnmanagedServices` is), you could do (I've probably got something wrong here):

```scala
private val cassandraStaticServices: Def.Initialize[Map[String, String]] = Def.setting {
  Map("cas_native" -> s"tcp://127.0.0.1:${lagomCassandrPort.value}/cas_native")
}
```

You can then easily modify this to also incorporate `lagomCassandraEnabled`.